server/service/src/sync/sync_buffer.rs

```diff
@@ -65,7 +65,7 @@ impl<'a> SyncBuffer<'a> {
                     .action(action.equal_to())
                     .integration_datetime(DatetimeFilter::is_null(true))
                     .source_site_id(match source_site_id {
-                        Some(site_id) => EqualFilter::equal_to_i32(site_id),
+                        Some(site_id) => EqualFilter::equal_any_or_null_i32(vec![site_id]),
```

Comment:
Need to integration `null` `source_site_id` on remote sites, as records from OMS Central still come through as null.
Should I also do something for OMS Central, I kind of think I should but wanted to get feedback first....

Reply:
I think this integration by 'source_site_id' was done for one use case, when omSupply Central has just received data as a push from omSupply remotes, and needs to integrate those records (just those, not the rest of the sync_buffer because the rest of the sync buffer might have unfinished sync batches). I don't think we need to change this

Reply:
I think we need to integrate Sync V5 and Sync V6 records together as one block. Should be safe to do Sync V5 then Sync V6 I think? Could be an option instead of doing this?